server/data_provider_canvas_esb.rb

```diff
@@ -124,38 +124,48 @@ def canvasAPICalendarEventsURL(uniqname, canvas_courses)
     full_request_url
   end
 
+  # Create a string that expands the array of arguments and concatenates the array prefix to each one
+  def self.urlArrayParameters(array_prefix, parameters)
+    parameters.inject("") { |result, parameter| result << "#{array_prefix}#{parameter}" }
+  end
+
   #  <canvas server>/api/v1/users/self/calendar_events
-  def canvasESBToDoLMS(uniqname, canvas_courses, security_file, esb_application)
+  def canvasESBToDoLMSArray(uniqname, canvas_courses, security_file, esb_application)
```

Comment:
Does the name of this method mean it returns an array?  It appears to return a `WAPIResultWrapper` object, not an array.